src/builtins/Transformers.jl

```diff
@@ -516,58 +518,50 @@ MLJBase.output_is_multivariate(::Type{<:OneHotEncoder}) = true
 
 ## Imputer
 """
-    fillImputer -   imputes by applying function to non missing values
+    FillImputer -   imputes by applying function to non missing values
                     default for continuous(median), CategoricalArray(most common),
                         Count (rounded median)
 
 
 """
-
-function commonCategoryFiller(vec::CategoricalArray)
-    ftab= freqtable(vec)
-    return names(ftab)[1][findmax(ftab)[2]]
-end
-
-function basetype(A::B) where {T,N,B<:AbstractArray{T,N}}
-    T
-end
-
-
-
-
-mutable struct fillImputer <: Unsupervised
+mutable struct FillImputer <: Unsupervised
     features::Vector{Symbol}
     continuous_fill::Union{Function}
     count_fill::Union{Function}
     categorical_fill::Union{Function}
     allowed_scitypes::Array{Any}
 
 end
```

Comment:
I suggest doing 

```julia
continuous_fill::CN
count_fill::CU
categorical_fill::CA   # or multiclass_fill::M
```
and adding `CN, CU, CA` as type parameters. 

How about `allowed_scitypes::Vector{Any}` or `Vector{Type}`?